xamarin/Xamarin.Forms
# [iOS] Fix initial color of RefreshControl in ListViews

### Description of Change ###
On iOS, if you make a ListView begin refreshing too soon after setting `RefreshControlColor`, the first refresh that occurs after setting the color won't properly reflect. This is actually a known bug in iOS: 

https://github.com/davbeck/RefreshTintFail/
https://github.com/uruly/RefreshControlTintColorDemo

This change applies the workaround mentioned in the above repos to Xamarin.Forms.

### Issues Resolved ### 
- fixes #5728 

### Platforms Affected ### 
- iOS

### Behavioral/Visual Changes ###
Users that are setting the refresh spinner color to one other than default color should now see that color reflected immediately. Currently, the refresh indicator may stay its _previous_ color if the refresh was performed very shortly after setting the color.

### Testing Procedure ###
Run the newly created reproduction screen in the core gallery. The refresh indicator should appear and be cyan, not black.

### PR Checklist ###
